Gemfile.template

```diff
@@ -38,7 +38,8 @@ gem "logstash-codec-json_lines"
 gem "logstash-codec-line"
 gem "logstash-codec-msgpack"
 gem "logstash-codec-multiline"
-gem "logstash-codec-netflow"
+# Pin to 3.x branch to avoid breaking change in Logstash 6.x
```

Comment:
We have changes in this codec only compatible with Logstash 7.x ?

Reply:
No not Logstash 7.x, but not BWC - I'll change the comment to be less ambiguous https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/CHANGELOG.md